src/appkit.cs

```diff
@@ -25201,4 +25769,278 @@ interface NSSliderAccessoryBehavior : NSCoding, NSCopying
+
+	[Mac (10,13)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface NSAccessibilityCustomRotorItemSearchDelegate
+	{
+		[Abstract]
+		[Export ("rotor:resultForSearchParameters:")]
+		[return: NullAllowed]
+		NSAccessibilityCustomRotorItemResult GetResult (NSAccessibilityCustomRotor rotor, NSAccessibilityCustomRotorSearchParameters searchParameters);
+	}
+
+	interface INSAccessibilityElementLoading {}
+
+	[Mac (10,13)]
+	[Protocol]
+	interface NSAccessibilityElementLoading
+	{
+		[Abstract]
+		[Export ("accessibilityElementWithToken:")]
+		[return: NullAllowed]
+		NSAccessibilityElement GetAccessibilityElement (INSSecureCoding token);
+
+		[Export ("accessibilityRangeInTargetElementWithToken:")]
+		NSRange GetAccessibilityRangeInTargetElement (INSSecureCoding token);
+	}
+
+	interface INSCollectionViewPrefetching { }
+
+	[Mac (10,13)]
+	[Protocol]
+	interface NSCollectionViewPrefetching
+	{
+		[Abstract]
+		[Export ("collectionView:prefetchItemsAtIndexPaths:")]
+		void PrefetchItems (NSCollectionView collectionView, NSIndexPath[] indexPaths);
+
+		[Export ("collectionView:cancelPrefetchingForItemsAtIndexPaths:")]
+		void CancelPrefetching (NSCollectionView collectionView, NSIndexPath[] indexPaths);
+	}
+
+	delegate bool DownloadFontAssetsRequestCompletionHandler (NSError error);
+
+	[Mac (10,13)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSFontAssetRequest : INSProgressReporting
+	{
+		[Export ("initWithFontDescriptors:options:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSFontDescriptor[] fontDescriptors, NSFontAssetRequestOptions options);
+
+		[Export ("downloadedFontDescriptors", ArgumentSemantic.Copy)]
+		NSFontDescriptor[] DownloadedFontDescriptors { get; }
+
+		[Export ("progress", ArgumentSemantic.Strong)]
+		NSProgress Progress { get; }
+
+		[Export ("downloadFontAssetsWithCompletionHandler:")]
+		void DownloadFontAssets (DownloadFontAssetsRequestCompletionHandler completionHandler);
+	}
+
+	[Category]
+	[BaseType (typeof(NSObject))]
+	interface NSObject_NSFontPanelValidationAdditions
+	{
+		[Export ("validModesForFontPanel:")]
+		NSFontPanelModeMask GetValidModes (NSFontPanel fontPanel);
+	}
+
+	[Mac (10, 13)]
+	[BaseType (typeof(NSObject))]
+	interface NSUserInterfaceCompressionOptions : NSCopying, NSCoding
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
+
+		[Export ("initWithCompressionOptions:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSSet<NSUserInterfaceCompressionOptions> options);
+
+		[Export ("containsOptions:")]
+		bool Contains (NSUserInterfaceCompressionOptions options);
+
+		[Export ("intersectsOptions:")]
+		bool Intersects (NSUserInterfaceCompressionOptions options);
+
+		[Export ("empty")]
+		bool Empty { [Bind ("isEmpty")] get; }
+
+		[Export ("optionsByAddingOptions:")]
+		NSUserInterfaceCompressionOptions ByAdding (NSUserInterfaceCompressionOptions options);
```

Comment:
That would be weird to use.

```
var a = new NSUserInterfaceCompressionOptions ("identifier");
var b = a.ByAdding (new NSUserInterfaceCompressionOptions ("otherIdentifier"));
```

Shouldn't it be `GetOptions` with an enum? Or `GetOptionsByAdding`...